inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -60,10 +60,53 @@ bool InLaneCruisingPlugin::onSpin()
   return true;
 }
 
+void InLaneCruisingPlugin::set_yield_client(ros::ServiceClient& client)
+{
+  yield_client_ = client;
+}
+
+bool InLaneCruisingPlugin::plan_trajectory_cb2(cav_srvs::PlanTrajectoryRequest& req,
+                                              cav_srvs::PlanTrajectoryResponse& resp)
+{
+
+  ROS_ERROR("in cb2");
+  if (yield_client_ && yield_client_.exists() && yield_client_.isValid()){
+    ROS_ERROR("Good Client");
+  }
+  cav_srvs::PlanTrajectory yield_srv;
+  if (req.initial_trajectory_plan.trajectory_id == "ILCReq"){
```

Comment:
Just checking, are these `trajectory_id`s with specific values only used for the unit testing? Nominally I believe our practice here is to use randomly-assigned GUIDs for trajectory ID's for traceability.